### DISCUSSION TITLE
Only one old (2021-03-28) position

### DISCUSSION THREAD

**Discussion by Ilpo55** [flobz/psa_car_controller#530] [Q&A]
{"altitude":84.900002,"error_altitude":2.5,"error_h_position":3.53,"error_v_speed":0.09,"fix_status":3,"heading":225.344299,"latitude":62.955273,"longitude":27.683623,"speed":0,"timestamp":"**2021-03-28T15:15:38Z**",....

Does anyone know why I get only one very old position and nothing after that? Or does someone get positions correctly?

**Comment by mrdave-75** [comment]
Same problem:
"latitude":44.631825,"longitude":11.008752,"speed":0,"timestamp":**"2022-10-05T06:24:04Z"**

**Reply by lvenditti** [reply]
Same problem to me. 
Position is update at 2023-03-10 10:06:36+00:00, as form "get_vehicleinfo":
 "last_position": {
    "type": "Feature",
    "geometry": {
      "coordinates": [
        13.xxxxx,
        42.xxxx,
        7x
      ],
      "type": "Point"
    },
    "properties": {
      "heading": 282,
      "signal_quality": 6,
      "type": "Acquire",
      "updated_at": "2023-03-10 10:06:36+00:00"
    }
  },
 
Instead the other information are up to date.
Most probably it is an Citroen issue, but into official MyCitroen APP the last position is update correctly.

Ciao

**Comment by Ilpo55** [comment]
MyCitroen APP does not read the position from the PSA server, while PSACC reads the postion form the PSA server.

(I have Peugeot with the same issue).

**Reply by lvenditti** [reply]
I have oppiste situation: MyCriteone APP read the position but PSACC not.

Could be that the server will send the position only 1 time and the first request is satified but not the following one ?

**Comment by Ilpo55** [comment]
>I have oppiste situation: MyCriteone APP read the position but PSACC not.

That is what I ment. 

I only tried to explain how it works:
PSACC reads from the server.
Myxxxxxx.app get the positions directly from the car by BT.
MYxxxxxx.app does not use sever to get the position.

**Reply by lvenditti** [reply]
so it seems that positions are no more available on server, right ?
I was not aware that MyxxxxAPP took positions from BT, are you sure?

Thanks

**Comment by Ilpo55** [comment]
>so it seems that positions are no more available on server, right ?

Not for us. Some users get the positions, some don't.

>I was not aware that MyxxxxAPP took positions from BT, are you sure?

Yes. You can do a test:
Leave your phone at home. Drive your car 500 m away and leave it there. Walk back to home. Can you see the position of your car in the app at home?
